src/styles/blocks/header.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  font-family: Roboto, sans-serif;
```

Comment:
u use this font in entire project. Set it globally in ur `reset.scss` file.
u can use `html` tag in ur case